.github/workflows/python-app.yml

```
@@ -37,7 +37,12 @@ jobs:
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
         pip install coverage flake8
         misc/download_language_models.sh
-    - name: Check errors with flake8
+    - name: Lint with flake8
+      run: |
+        # Lint with deactivated whitespace before/after operator check, and deactivated line length for unit tests file
+        # TODO: Config file
+        flake8 . --max-line-length=120 --show-source --statistics "--ignore=E221,E222,E272,E211,E202,E241" --exclude="./protorepo/*" --per-file-ignores="./tests_recommendations.py:E501"    
+      - name: Check errors with flake8
```

I think there's a tabulation error here.
```suggestion
    - name: Check errors with flake8
```